src/liveEditor/components/emptyBlock.tsx

```diff
@@ -0,0 +1,43 @@
+import { CslpData } from "../../cslp/types/cslp.types";
```

Comment:
Can you attach a screenshot of the rendered empty block? I'd love to see how it looks.

Reply:
Sure, hyg
<img width="1512" alt="Screenshot 2024-05-08 at 2 59 56 PM" src="https://github.com/contentstack/live-preview-sdk/assets/155638868/08a5321d-a5fa-4fa8-b937-587bde033da1">